mkdocs/docs/HPC/jupyter.md

```diff
@@ -0,0 +1,129 @@
+# Jupyter notebook
+
+## What is a Jupyter notebook
+
+A [Jupyter notebook](https://jupyter.org/) is an interactive, web-based environment that allows you to create documents that contain live code, equations, visualizations, and plaintext. The code blocks in these documents can be used to write Python, Java, R and Julia code, among others. The combination of code executions with text and visual outputs make it a useful tool for data analysis, machine learning and educational purposes.
+
+## Using Jupyter Notebooks on the HPC
+
+### Starting a notebook using the web portal
+
+Through the web portal you can easily start a Jupyter notebook on a workernode, via the *Jupyter Notebook* button under the *Interactive Apps* menu item.
+
+<center>
+![image](img/ood_start_jupyter.png)
+</center>
+
+### Importing libraries
+
+Importing libraries in a notebook running on the HPC isn't as straight forward as on notebooks running on your local machine. You can only import libraries that are part of certain modules on the HPC. To find the right module, first look at the toolchain used by the Jupyter notebook version. This can be found when looking at the `JupyterNotebook version` field when creating a new Jupyter notebook session. In the image above `7.2.0` is the notebook version and `GCCcore 13.2.0`is the toolchain used.
```

Comment:
This should be reworded a bit.

People can actually install additional Python packages themselves to use in a Jupyter notebook, even from within the notebook itself, but it gets tricky fast.

Another approach could be to first set up a Python virtual environment on top of the centrally installed modules, and then activate that through the `Custom code` block so the notebook can load the Python packages installed in the virtual environment (see also #639).

Reply:
You should also use a specific example as a use case, like using `numpy` in a notebook, and then explaining step by step which steps to take to set things up:

1) Figure out which extra modules need to be loaded (`SciPy-bundle` for `numpy`), and how to make sure that they're compatible with the Jupyter module that is being used (which can be done by trying to load those modules together in a terminal session);
2) Specifying those `module load` statements in the `Custom code` block before starting the Jupyter notebook session;

We should also provide some troubleshooting advice: if the Jupyter notebook session fails to start, go check the `output.log` file that's available in the session directory for clues, there may be module load conflict error messages in there (give an example so people know what to look for)